homeassistant/components/mailbox/demo.py

```diff
@@ -31,10 +31,12 @@ def __init__(self, hass, name):
         """Initialize Demo mailbox."""
         super().__init__(hass, name)
         self._messages = {}
+        txt = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. "
         for idx in range(0, 10):
             msgtime = int(dt.as_timestamp(
                 dt.utcnow()) - 3600 * 24 * (10 - idx))
-            msgtxt = "This is recorded message # %d" % (idx)
+            msgtxt = ("Message %d. %s" %
```

Comment:
Please use the string format function instead '%' -> `"Bla {} is {}".format(1, 2)`